draft-ietf-quic-transport.md

```diff
@@ -2511,11 +2531,10 @@ mentions a stream ID.  In this state, the endpoint MUST observe advertised
 stream and connection data limits (see {{flow-control}}).
 
 A stream transitions from this state to "closed" by completing transmission of
-all data.  This includes sending all data carried in STREAM frames up including
+all data.  This includes sending all data carried in STREAM frames including
 the terminal STREAM frame that contains a FIN flag.
 
-A stream becomes "closed" when the endpoint sends and receives acknowledgment of
-a RST_STREAM frame.
+A stream also becomes "closed" when the endpoint sends a RST_STREAM frame.
```

Comment:
Looks like this text came in as part of #459, but it's inconsistent with the text in "open" which says that RST_STREAM causes transitions immediately on send, which I thought is what we'd agreed.  I'm going to leave it for now until @martinthomson can weigh in.